.github/workflows/python.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Distribute Python choreolib
+
+permissions:
+  id-token: write # IMPORTANT: this permission is mandatory for trusted publishing
+
+on:
+  push:
+    branches: [master]
+    tags:
+      - "v*"
+  pull_request:
+    branches: [master]
+
+jobs:
+  buildAndDeploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          sparse-checkout-cone-mode: false
+          fetch-tags: true
+          fetch-depth: 99999
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel pytest
+
+      - name: Build wheel
+        working-directory: ./choreolib/py
+        run: |
+          python setup.py sdist bdist_wheel
+
+      - name: Run Unit Tests
+        working-directory: ./choreolib/py
+        run: |
+          pip install --no-cache-dir dist/*.whl
+          pytest
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@master
+        with:
+          name: dist
+          path: ./choreolib/py/dist/
+
+      - name: Publish package distributions to PyPI
+        # Only upload on tags
+        if: startsWith(github.ref, 'refs/tags/v')
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          packages_dir: ./choreolib/py/dist/
+
+    permissions:
+      id-token: write # IMPORTANT: this permission is mandatory for trusted publishing
```

Comment:
Doubled?

Reply:
I didn't actually need this permissions line for Sleipnir's Python package publishing.